debian/rules

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/make -f
+
+HANAME := $(lastword $(shell grep "Package: pg-auto-failover-cli" debian/control))
```

Comment:
Do we keep HANAME here as a variable name?

Reply:
named it CLINAME now